src/audio/module_adapter/module_adapter_ipc3.c

```diff
@@ -75,6 +75,7 @@ int module_adapter_init_data(struct comp_dev *dev,
 	case SOF_COMP_SELECTOR:
 	case SOF_COMP_DEMUX:
 	case SOF_COMP_MUX:
+	case SOF_COMP_MIXER:
```

Comment:
Thanks for finding this but this isn't correct. We need to add a new case for mixer that skips component specific data parsing. See [here](https://github.com/thesofproject/sof/blob/main/src/ipc/ipc3/helper.c#L308)